tests/format/vue/multiparser/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -70,6 +70,60 @@ export default {
 ================================================================================
 `;
 
+exports[`lang-ts-multiple-script-tags.vue format 1`] = `
+====================================options=====================================
+parsers: ["vue"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<script></script>
+<script></script>
+<script></script>
+<script></script>
+<script></script>
+<script></script>
+<script setup lang="ts">
+let x:      string |         number = 1
+</script>
+<script></script>
+<script></script>
+<script></script>
+<template>
+  <span
+  v-if=" (x    as string).length  >     0"
+  v-for="a in [1,2,  3,4,5].map(   (x  :    number) => x *   x)"
+  :foo="  (x    as number).toFixed(   2) "
```

Comment:
These aren't formatted in this branch due to https://github.com/prettier/prettier/issues/14432 .
Rebasing to next branch, they are formatted.
Feel free to ask me to
- remove these lines
- or, change base branch

Reply:
We can ignore it.